build-gluster-org/jobs/centos8-regression.yml

```diff
@@ -6,13 +6,7 @@
     concurrent: true
 
     scm:
-    - git:
```

Comment:
No renaming of the job name ?

Reply:
I have renamed only the jobs which are a part of smoke and regression jobs, i.e which gets triggered on PR. centos8-regression is not a part of the development workflow, so I chose to keep it as it is.